vscode-lean4/src/leanclient.ts

```diff
@@ -478,7 +480,10 @@ export class LeanClient implements Disposable {
     configChanged(e : ConfigurationChangeEvent): void {
         let newToolchainPath = this.storageManager.getLeanPath();
         if (!newToolchainPath) newToolchainPath = toolchainPath();
-        if (this.toolchainPath !== newToolchainPath){
+        if (this.toolchainPath === undefined){
+            this.toolchainPath = newToolchainPath;
```

Comment:
Could you put a comment explaining why we don't need to restart in this case?

Reply:
I pushed a different fix, I was seeing vs raise the configuration change before anything else, before restart had ever been called once, so this.toolchainPath was uninitialized, but I pushed a change now that initializes it in the constructor which is a better way to solve this problem.